### PR TITLE
Configure automated testing in Travis

### PR DESCRIPTION
**Pull request by paulfurley**
So that pull requests etc are automatically tested.

(You'll need to visit https://travis-ci.org to actually enable the build hook)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+  - "3.3"
```
Comment (StevenMaude) [inline .travis.yml]: Should 3.3 be listed here (since `scraperwiki-python` is blocking it).

Otherwise, looks good!

**Comment by StevenMaude** [comment]
@paulfurley If you can bump this so 3.3 is changed to 3.4, I'll merge it.

**Comment by paulfurley** [comment]
Done :)

**Comment by StevenMaude** [comment]
Thanks! Just need someone to actually enable Travis; I'm not sure how I'd get access to do that @drj11 @scraperdragon.

**Pull request merged**